unstract/core/src/unstract/core/data_models.py

```
@@ -0,0 +1,2278 @@
+
+
+# File Operation Constants
+class FileOperationConstants:
+    """Constants for file operations."""
+
+    READ_CHUNK_SIZE = 4194304  # 4MB chunks for file reading
+    MAX_RECURSIVE_DEPTH = 20  # Maximum directory traversal depth
+    DEFAULT_MAX_FILES = 100  # Default maximum files to process
+
+    # File pattern defaults
+    DEFAULT_FILE_PATTERNS = ["*"]
```

Ruff (0.13.1)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)